src/components/DataLinkTable/DataLinkTable.test.tsx

```diff
@@ -3,5 +3,5 @@ import { shallow } from 'enzyme';
 import DataLinkTable from './DataLinkTable';
 
 it('Renders without crashing', () => {
-  shallow(<DataLinkTable />);
+  shallow(<DataLinkTable id="testje" />);
```

Comment:
This can be removed I think? Or change the id.

Reply:
`id` is a required prop. It's a test so the value for this field doesn't really matter.